### PR TITLE
[SPM] Migrating from pod to spm + bumping tag to 0.2.0

### PR DESCRIPTION
**Pull request by Muhieddine-El-Kaissi**
**Installation**
To integrate TTCalendarPicker into your Xcode project using Swift Package Manager, add it to the dependencies value of your `Package.swift`:

```swift
dependencies: [
    .package(url: "https://github.com/thumbtack/TTCalendarPicker.git", .upToNextMajor(from: "0.2.0"))
]
```

**Test passes**

Screenshot 2023-04-18 at 11 30 46 AM

**Example app runs**

Simulator Screen Shot - iPhone 8 - 2023-04-18 at 11 39 43
Simulator Screen Shot - iPhone 8 - 2023-04-18 at 11 39 37

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@ profile
 DerivedData
 *.hmap
 *.ipa
+Package.resolved
```
Comment (scottasoutherland) [inline .gitignore]: I guess we don't check in the package.resolved because this is a library, not a final target ?

**Review by dwroth — APPROVED** [review]
Love seeing all that removed code 💪

File: README.md [hunk 2]
```diff
@@ -16,22 +12,28 @@ don't plan on using within our own apps, we will be actively fixing bugs as they
...
-To run the example project, clone the repo, and run `pod install` from the Example directory first.  Then build and run the 
-TTCalendarPicker-Example project.
+To run the example project, clone the repo then build and run the TTCalendarPicker-Example project.
 
 ## Requirements
 
-- iOS 10.0+
-- Xcode 10.0+
-- Swift 4.0+
+- iOS 13.0+
```
Comment (scottasoutherland) [inline README.md:19]: Does this really work back on iOS 13 ?

**Review by scottasoutherland — COMMENTED** [review]
Generally LGTM, but i don't know this library very well. Will let someone else weigh in.

**Comment by ileitch** [comment]
Hey guys 👋🏻, apparently I'm a watcher on this repo still. Looks like you're migrating from CocoaPods to SPM, that's awesome 😎. I hope you're all going great!

File: README.md (hunk 2 above)
Comment (Muhieddine-El-Kaissi) [reply README.md:19]: yes, the minimum deployment target in project is ios13

File: .gitignore (hunk 1 above)
Comment (Muhieddine-El-Kaissi) [reply .gitignore]: it can be both as we specify the exact version but thinking about it again it seems to be better to check it in

**Pull request merged**